portfolio/src/main/webapp/script.js

```diff
@@ -75,8 +75,21 @@ function addRandomProject() {
   linkElement.href = link.url;
 }
 
-function appendData() {
-    fetch("/data").then( response => response.text() ).then ( data => {
-        document.body.innerHTML += data;
-    });
+/**
```

Comment:
Please standardize indenting here.  Rest of file looks to be 2 spaces indent, this section is 4.